ios/Podfile.lock

```diff
@@ -1,57 +1,90 @@
+    - GoogleToolboxForMac/Defines (= 2.1.4)
+  - nanopb (0.3.8):
+    - nanopb/decode (= 0.3.8)
+    - nanopb/encode (= 0.3.8)
+  - nanopb/decode (0.3.8)
+  - nanopb/encode (0.3.8)
+  - Protobuf (3.5.0)
+  - React (0.55.4):
+    - React/Core (= 0.55.4)
   - react-native-background-timer (2.0.0):
     - React
-  - React/Core (0.53.3):
-    - yoga (= 0.53.3.React)
-  - yoga (0.53.3.React)
+  - React/Core (0.55.4):
+    - yoga (= 0.55.4.React)
+  - RNKeychain (3.0.0-rc.3):
+    - React
+  - yoga (0.55.4.React)
 
 DEPENDENCIES:
-  - FirebaseMessaging
+  - Firebase/Core
+  - Firebase/Messaging
   - React (from `../node_modules/react-native`)
   - react-native-background-timer (from `../node_modules/react-native-background-timer`)
+  - RNKeychain (from `../node_modules/react-native-keychain`)
   - yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
+SPEC REPOS:
+  https://github.com/cocoapods/specs.git:
+    - Firebase
+    - FirebaseAnalytics
+    - FirebaseCore
+    - FirebaseInstanceID
+    - FirebaseMessaging
+    - GoogleToolboxForMac
+    - nanopb
+    - Protobuf
+
 EXTERNAL SOURCES:
   React:
-    :path: ../node_modules/react-native
+    :path: "../node_modules/react-native"
   react-native-background-timer:
-    :path: ../node_modules/react-native-background-timer
+    :path: "../node_modules/react-native-background-timer"
+  RNKeychain:
+    :path: "../node_modules/react-native-keychain"
   yoga:
-    :path: ../node_modules/react-native/ReactCommon/yoga
+    :path: "../node_modules/react-native/ReactCommon/yoga"
 
 SPEC CHECKSUMS:
-  FirebaseAnalytics: 4d7040fefc3cd8b291cde35f12cf063d7963f15d
-  FirebaseCore: 1a8bf6c795ad07d2918278657954ffd2552e9c17
-  FirebaseInstanceID: 9fbf536668f4d3f0880e7438456dabd1376e294b
-  FirebaseMessaging: 227406c05b0dc9290702d2e9f18ab5528f0c2cf2
-  GoogleToolboxForMac: 8e329f1b599f2512c6b10676d45736bcc2cbbeb0
-  Protobuf: d582fecf68201eac3d79ed61369ef45734394b9c
-  React: fd20e9486c9d994fb6e40d6e67dbd29c7709889c
-  react-native-background-timer: 10063c04bf85d7f8811dff8c74399f0aa715245f
-  yoga: c7c8e64dbc2552b1ceac0b491ff03f5e5128b501
+  Firebase: d6861c2059d8c32d1e6dd8932e22ada346d90a3a
+  FirebaseAnalytics: 19812b49fa5f283dd6b23edf8a14b5d477029ab8
+  FirebaseCore: cafc814b2d84fc8733f09e653041cc2165332ad7
+  FirebaseInstanceID: 83e0040351565df711a5db3d8ebe5ea21aca998a
+  FirebaseMessaging: f2360a966ecfb0d14facf0fbdf306efc2df0ddbe
+  GoogleToolboxForMac: 91c824d21e85b31c2aae9bb011c5027c9b4e738f
+  nanopb: 5601e6bca2dbf1ed831b519092ec110f66982ca3
+  Protobuf: 8a9838fba8dae3389230e1b7f8c104aa32389c03
+  React: aa2040dbb6f317b95314968021bd2888816e03d5
+  react-native-background-timer: 63dcbf37dbcf294b5c6c071afcdc661fa06a7594
+  RNKeychain: 627c6095cef215dd3d9804a9a9cf45ab96aa3997
+  yoga: a23273df0088bf7f2bb7e5d7b00044ea57a2a54a
 
-PODFILE CHECKSUM: 653ad2d673bf8aa8127325a078e07aedc8dbe463
+PODFILE CHECKSUM: 018eba4ceb8d7d516fe0130fab47bded597e9d44
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.5.3
```

Comment:
Is this required?

Reply:
yes it is

Reply:
Firebase deps require higher version of cocoapods